fury/actor.py

```diff
@@ -444,6 +444,45 @@ def contour_from_roi(data, affine=None,
     return skin_actor
 
 
+def contour_from_label(data, affine=None,
+                       color=np.array([1, 0, 0]), opacity=1):
+    """Generate surface actor from a binary labeled Array.
+
+    The color and opacity of the surface can be customized.
+
+    Parameters
+    ----------
+    data : array, shape (X, Y, Z)
+        A labeled array file that will be binarized and displayed.
+    affine : array, shape (4, 4)
+        Grid to space (usually RAS 1mm) transformation matrix. Default is None.
+        If None then the identity matrix is used.
+    color : (1, 3) ndarray
+        RGB values in [0,1].
+    opacity : float
+        Opacity of surface between 0 and 1.
+
+    Returns
+    -------
+    tuple of contour_assembly : vtkAssembly
+        Tuple of Array surface object displayed in space
+        coordinates as calculated by the affine parameter
+        in the order of their roi ids.
+    """
+    unique_roi_surfaces = []
+
+    unique_roi_id = np.unique(data)
+
+    for roi_id in unique_roi_id:
+        if roi_id != 0:
+            roi_surface = np.isin(data, roi_id)
+            roi_surface = roi_surface * 1
+            roi_surface = contour_from_roi(roi_surface, affine, color, opacity)
+            unique_roi_surfaces.append(roi_surface)
+
+    return tuple(unique_roi_surfaces)
```

Comment:
This is a good idea!  unfortunately, we would like to return only 1 actor and not a list of actors. So your solution is not valid.
Look at the code of `contour_from_roi`and I think you should update this part: 

```python
skin_extractor = vtk.vtkContourFilter()
skin_extractor.SetInputData(image_resliced.GetOutput())

skin_extractor.SetValue(0, 1)
skin_extractor.SetValue(1, `your_label_value`)
skin_extractor.SetValue(2, `your_label_value_2`)
.....
```

Reply:
for the first argument of `SetValue()` I pass the ID of the contour, but what do I pass for the second argument? Do I need to take another parameter to store the labels?

Also, the vtk [documentation](https://vtk.org/doc/nightly/html/classvtkContourFilter.html#aa33aa2585c805a22bfa9b8454d69f9a1) says that the second argument should be a value.